cluster-autoscaler/cloudprovider/ionoscloud/cache.go

```diff
@@ -18,28 +18,18 @@ package ionoscloud
 
 import (
 	"sync"
-	"time"
 
 	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider"
 	"k8s.io/klog/v2"
 )
 
-const nodeGroupCacheEntryTimeout = 2 * time.Minute
-
-type nodeGroupCacheEntry struct {
-	data cloudprovider.NodeGroup
-	ts   time.Time
-}
```

Comment:
Pls gimme some background on this. This looks to me as if we no longer require to refresh any cached node pools after 2m. Why?